scripts/Tools/clean_build

```diff
@@ -86,7 +86,13 @@ if( $cnt==0){
     $opts{glc} = 1;
     $opts{ice} = 1;
     $opts{rof} = 1;
-    # Do not clean shared land library in a test case unless explicitly requested
```

Comment:
I find this explanation confusing but I'm not sure how to improve on it.  
The clm doesn't need to be cleaned because if it's dependencies change
the  shared library path changes and either another clm library is there or will be built there.